pkg/repository/metered/metered.go

```diff
@@ -32,7 +32,7 @@ func NewMetered(entitlements repository.EntitlementsRepository, l *zerolog.Logge
 
 var ErrResourceExhausted = fmt.Errorf("resource exhausted")
 
-func MakeMetered[T any](ctx context.Context, m *Metered, resource dbsqlc.LimitResource, tenantId string, f func() (*string, *T, error)) (*T, error) {
+func MakeMetered[T any](ctx context.Context, m *Metered, resource dbsqlc.LimitResource, tenantId string, numberOfResources int32, f func() (*string, *T, error)) (*T, error) {
```

Comment:
@grutt I needed to modify the metering because it was all based on 1 resource (from my reading of it) - modified it to now take a number of resources we want to meter. Tested it by hardcoding enforce limits -seems to work.

Reply:
ahh i thought i thought ahead here... i probably did think ahead but didnt implement ahead.